nav2_simple_commander/launch/assisted_teleop_example_launch.py

```diff
@@ -0,0 +1,77 @@
+# Copyright (c) 2022 Joshua Wallace
```

Comment:
This is sufficiently derivative you should include the original copyright person for the launch file you copied this from. You can also have your name with 2022 after too.

Reply:
The launch files are pretty much the same except the node being launched. We can probably refactor this at some point.